packages/contact-manager-rest-api/src/contact-manager-api-server.ts

```diff
@@ -0,0 +1,85 @@
+import { agentContext } from '@sphereon/ssi-sdk.core'
+import { TAgent } from '@veramo/core'
+
+import express, { Express, Router } from 'express'
+import { ContactManagerMRestApiFeatureEnum, IContactManagerAPIEndpointOpts, IRequiredPlugins } from './types'
+import {
+  identityReadEndpoints,
+  partyReadEndpoints,
+  partyTypeReadEndpoints,
+  partyWriteEndpoints
+} from './api-functions'
+import { copyGlobalAuthToEndpoints, ExpressSupport } from '@sphereon/ssi-express-support'
+
+export class ContactManagerApiServer {
+  private readonly _express: Express
+  private readonly _agent: TAgent<IRequiredPlugins>
+  private readonly _opts?: IContactManagerAPIEndpointOpts
+  private readonly _router: Router
+
+  constructor(args: { agent: TAgent<IRequiredPlugins>; expressSupport: ExpressSupport; opts?: IContactManagerAPIEndpointOpts }) {
+    const { agent, opts } = args
+    this._agent = agent
+    copyGlobalAuthToEndpoints({ opts, keys: ['contactRead', 'contactTypeRead', 'identityRead'] })
```

Comment:
This should include all potential configs used by the endpoints. Since you enabled write endpoints, it should also include these configs

Reply:
changed to include all